index.js

```diff
@@ -60,7 +61,7 @@ class DHT extends EventEmitter {
     this._onrow = (row) => row.on('full', (node) => this._onfullrow(node, row))
     this._nonePersistentSamples = []
     this._bootstrapping = this._bootstrap()
-    this._bootstrapping.catch(noop)
```

Comment:
this is the only one that is relevant for safety catching actually (the rest of are not try-catches), lets just keep it as is